examples/exactly-once/src/main/java/com/example/myapexapp/Application.java

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
@amberarrow there are multiple commits, please look at the net diff. (CI would fail if license header was missing.)